### ISSUE TITLE
cannot select characters in slots 8+

### ISSUE THREAD

**Issue by Tokyo-All-Ready** [kolton/d2bot-with-kolbot#486]
you cannot scroll down and click on characters if they are in slots 9 -> 16

d2bs does not scroll down to select a character if past slot 8

**Comment by noah-** [comment]
yes, that is a current limitation, the starters need to be modified to account for additional slots

**Comment by ross-weir** [comment]
This is a gross one. There will need to be a key press function added to d2bs. There are no down or up arrows to click in character select screen on bnet. Unless there is some other way of scrolling other than the UP and DOWN arrows.

Using `0x19` `realmpacket` events I can find the location of certain characters, but if they are on a lower page there's no way of scrolling down currently.

`ControlAction.getCharacters` could also be replaced with using `0x19` for getting a complete character count as `ControlAction.getCharacters` doesn't get characters that aren't in the current view.

Further adding to the grossness is if there is scrolling needed to be done and there isn't 2 slots side-by-side occupied with characters you cannot scroll down on the side that's missing a slot. So this will have to be considered to make sure it works properly in this case.

**Comment by justinpees** [comment]
I tried fixing this once and i still think that it possible to add into the script.

right after the part where it scans the 8 first player slots names' of characters for a match, you want to add an if statement: if no match, click once on player slot 6 and scroll/downarrow (4 times). then recall the part where it scans the character screen for a matching character name.

problem was i had no idea how to position a click like they do and force a down arrow key.

if anyone knows how to do it, please let me know. this should be an easy fix im surprised it wasnt the first to get fixed in update
